py/src/braintrust/oai_wrappers/non_proxy_wrappers.py

```diff
@@ -0,0 +1,180 @@
+"""Wrappers for the openAI client which do not go through the Braintrust
```

Comment:
This is purely a cut-and-paste from `oai.py`, except for changing the names of the wrapper classes.